apps/mobile/src/components/add-wallet/add-wallet-sheet.tsx

```diff
@@ -83,63 +58,25 @@ function getUnavailableFeatures(theme: Theme) {
   return UNAVAILABLE_FEATURES;
 }
 
-interface AddWalletModalBaseProps {
-  addWalletModalRef: RefObject<BottomSheetModal>;
+interface AddWalletSheetBaseProps {
+  addWalletSheetRef: RefObject<SheetRef>;
 }
 
-export function AddWalletModal({ addWalletModalRef }: AddWalletModalBaseProps) {
-  const notifyUserModalRef = useRef<BottomSheetModal>(null);
-  const router = useRouter();
-  const [optionData, setOptionData] = useState<OptionData | null>(null);
-  const createWallet = useCallback(() => {
-    router.navigate(APP_ROUTES.WalletCreateNewWallet);
-    addWalletModalRef.current?.close();
-  }, [router]);
-
-  const restoreWallet = useCallback(() => {
-    router.navigate(APP_ROUTES.WalletRecoverWallet);
-    addWalletModalRef.current?.close();
-  }, [router]);
-
-  function onOpenNotificationsModal(option: OptionData) {
-    setOptionData(option);
-    notifyUserModalRef.current?.present();
-  }
-  function onCloseNotificationsModal() {
-    setOptionData(null);
-  }
-
-  return (
-    <>
-      <AddWalletModalUI
-        onOpenNotificationsModal={onOpenNotificationsModal}
-        createWallet={createWallet}
-        restoreWallet={restoreWallet}
-        addWalletModalRef={addWalletModalRef}
-      />
-
-      <NotifyUserModal
-        optionData={optionData}
-        onCloseNotificationsModal={onCloseNotificationsModal}
-        notifyUserModalRef={notifyUserModalRef}
-      />
-    </>
-  );
-}
-interface AddWalletModalUIProps extends AddWalletModalBaseProps {
+interface AddWalletSheetUIProps extends AddWalletSheetBaseProps {
   createWallet(): unknown;
   restoreWallet(): unknown;
   onOpenNotificationsModal(option: OptionData): unknown;
 }
-export function AddWalletModalUI({
-  addWalletModalRef,
+export function AddWalletSheetUI({
```

Comment:
I wonder if we should use Layout suffix as with extension?

Reply:
I can change it. @fbwoolf - I'll start using that pattern going forward to match with the `extension` as you suggested